code/game/objects/effects/contraband.dm

```diff
@@ -650,6 +650,11 @@
 	desc = "I told you to shake it, no stirring."
 	icon_state = "poster32_legit"
 
+/obj/structure/sign/poster/official/marvin
+	name = "Marvin"
+	desc = "WANTED: Marvin is a escaped martian from space station 8 please report any sighting of him to centcom officials"
```

Comment:
desc = "WANTED: Marvin is a**n** escaped martian from space station 8**. P**lease report any sighting of him to centcom officials**.**"

<3